code/modules/mob/living/carbon/human/species.dm

```diff
@@ -921,6 +924,38 @@
 		else
 			H.throw_alert("nutrition", /obj/screen/alert/starving)
 
+/datum/species/proc/handle_disgust(mob/living/carbon/human/H, var/metabolism_rate)
```

Comment:
I originally did this the same as how nutrition is being handled but I'll move the proc over to the stomach